src/styles/main.scss

```
@@ -1,3 +1 @@
-body {
-  margin: 0;
-}
+@import "header", "navigation", "container", "card", "reset";
```

Partials should be in separated lines:
```suggestion
@import 
  "header", 
  "navigation", 
  "container",
  "card", 
  "reset";
```